lib/origen_testers/atp/flow.rb

```diff
@@ -337,8 +337,11 @@ def test(instance, options = {})
 
           name = (options[:name] || options[:tname] || options[:test_name])
           unless name
-            [:name, :tname, :test_name].each do |m|
-              name ||= instance.respond_to?(m) ? instance.send(m) : nil
+            # Starting in Ruby3 type Symbol responds to name
```

Comment:
@pderouen thanks for the explanation.  

So Ruby < 3 if `instance` was a Symbol then `name` would end up becoming `nil` (since nothing responds)?   Guess I'm not clear on the original behavior.

Reply:
You are correct. With Ruby < 3, <code>name</code> would end up being <code>nil</code> when <code>instance</code> is a Symbol.

This update ensures the same behavior with Ruby >= 3.